File: java/salario.java
import java.util.Scanner;
public class salario {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int number = sc.nextInt();
        int salaperhour = sc.nextInt();
        double valorperhour = sc.nextDouble();
        
        System.out.println("NUMBER = " + number);
        String str = String.format("%.2f",salaperhour * valorperhour);
        System.out.println("SALARY = U$ " + str);
        
        
        sc.close();
    }
}

File: cpp/salario.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main() {
    int number;
    cin >> number;
    int salaperhour;
    cin >> salaperhour;
    double valorperhour;
    cin >> valorperhour;
    cout << "NUMBER = " << number << endl;
    cout << "SALARY = U$ " << fixed << setprecision(2) << salaperhour * valorperhour << endl;
    return 0;    
}
